landlab/utils/source_tracking_algorithm.py

```diff
@@ -0,0 +1,234 @@
+
+"""
+26 Jan 2016
+last_edited: 08 Jun 2017
+Authors: Sai Nudurupati & Erkan Istanbulluoglu
+
+Additional Reference: source_tracking_algorithm_user_manual.
+
+MD - Modeling Domain
+HSD - Hydrologic Source Domain
+"""
```

Comment:
This would be a great place to add text that describe what the MD and HSD are. Defining them as you do here is a good start but not sufficient.